### DISCUSSION TITLE
AI學習公開資料到底有沒有錯？

### DISCUSSION THREAD

**Discussion by Raiden1121** [1101-GS4719-C/Python-Community-GS4719-C#2046] [General]
https://technews.tw/2024/08/10/nvidias-ai-team-reportedly-scraped-youtube-netflix-videos-without-permission/

相信大家都知道AI是需要大量資料來學習的，像前陣子很紅的AI繪圖也是一大爭議。文中這句「AI學習同於個人權利，亦即有權『從其他來源學習事實、想法、資料或資訊，並表達自己的觀點』」讓我覺得很直得討論。我也比較認同中Nvidia的想法，因為任何人也是利用公開資訊學習再結合自己的想法來創作，因此我認為AI學習「公開」的資料並沒有錯。

你是否認為AI有學習公開資料的權利？歡迎討論

**Comment by ds0w0** [comment]
我認為AI學習「公開」的資料確實沒有錯，但就像是高度模仿他人的作品會被為人詬病一樣，並拿AI作品來商業化、說自己是原創的行為就存在問題了。有權利不代表能侵犯他人的權益，就像是論文要附上參考資料、或是同人二創要符合創用CC等條款一樣，被AI拿來當素材學習的內容應該被標註以尊重原創作者。

**Comment by Nana1218** [comment]
我覺得AI公開學習資料是沒有錯的，若只是做為學習用途，精進自己是可以的，但不應該將AI作為營利的工具。

**Comment by kingsophy** [comment]
我認為AI學習公開的資料沒有錯，AI通過學習大量的公開資料，能夠進行創新，為社會帶來更多便利和進步。限制AI的學習可能會阻礙技術發展，減少潛在的創新。

**Comment by hunteryang0108** [comment]
以商業的角度來看，文本的原作者確實很有可能被喜歡自己的資料被無償運用，雖然AI發展的公益價值看似遠大於原作者的個人利益，但如果不是這些人持續創作，AI也不會有那麼多資料可以訓練。因此我認為需要有一個合理的補償機制補償內容提供者。
